iosApp/Picover.xcodeproj/project.pbxproj

```diff
@@ -406,6 +432,11 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
+				OTHER_LDFLAGS = (
+					"$(inherited)",
+					"-framework",
+					shared,
+				);
```

Comment:
That's the result of point 6 from the guide. 
Unfortunately, adding value `$(inherited) -framework shared` via Xcode result in 
```
OTHER_LDFLAGS = (
					"$(inherited)",
					"-framework",
					shared,
				);
```